js/index.js

```diff
@@ -16,4 +16,132 @@ for (i= 0; i < skills.length; i++){
     let skill = document.createElement('li');
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+
+
+//handle message form submit
+let messageForm = document.getElementsByName("leave_message");
+
+console.log(messageForm[0], "this is the form");
+
+
+
+messageForm[0].addEventListener("submit", function(event){
+    event.preventDefault();
+
+    let theMessageForm = event.target;
```

Comment:
This isn't used: Either delete, or use it to shorten the assignments below.

Reply:
@mccalluc okay on my computer I fixed the reset button so instead of saying event.target.reset() it now says messageForm.reset()